src/work-allocation/containers/task-action/task-action-container.component.ts

```diff
@@ -5,9 +5,9 @@ import { ConfigConstants } from '../../components/constants';
 import { InfoMessage, InfoMessageType, TaskActionType, TaskService, TaskSort } from '../../enums';
 import { InformationMessage } from '../../models/comms';
 import { TaskFieldConfig, TaskServiceConfig } from '../../models/tasks';
-import { InfoMessageCommService, WorkAllocationTaskService } from '../../services';
+import { InfoMessageCommService, WorkAllocationTaskService, CaseworkerDataService } from '../../services';
```

Comment:
These are not alphabetical and will cause a lint warning.

Reply:
removed it completely as its not even required as per below comment. caseworkers coming from snapshot